test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageExtractorTests.cs

```diff
@@ -2670,7 +2670,7 @@ var pathToAFrDllInRunTimePackage
                 var expectedVerifierSettings = new SignedPackageVerifierSettings(
```

Comment:
Should this tests also use `GetAcceptModeDefaultPolicy(...)` and `GetRequireModeDefaultPolicy(...)` instead of having a custom settings object?